package.json

```diff
@@ -28,7 +28,7 @@
     "start-stepper": "cd packages/stepper && yarn start",
     "start-forms": "cd packages/forms && yarn start",
     "start-theme": "cd packages/theme && yarn start",
-    "publish": "lerna publish --force-publish=*",
+    "publish": "lerna publish --force-publish=* --no-verify-access",
```

Comment:
Since it's temporary, we can't use this instead?
```bash
$> yarn publish -- --no-verify-access
```

Reply:
Hey that's really clever :D 
I close the PR

Reply:
I initiated the PR because I changed the lerna.json to disable that, but it didn't work, and I found the cli option that worked. But yes it doesn't worth a PR for that then